contrib/drivers/openstack/cinder/cinder.go

```diff
@@ -22,6 +22,9 @@ Go SDK.
 package cinder
 
 import (
+	"io/ioutil"
```

Comment:
Please use `go fmt' to improve the code style.

Reply:
I have formatted the file，Thank you for your opinions